sygnal/gcmpushkin.py

```diff
@@ -113,10 +114,35 @@ def __init__(self, name, sygnal, config, canonical_reg_id_store):
 
         tls_client_options_factory = ClientTLSOptionsFactory()
 
-        self.http_agent = Agent(
+        http_proxy_url = None
+        http_proxy_creds = None
+
+        # use the Sygnal global proxy configuration
+        proxycfg = sygnal.config["proxy"]
+
+        if proxycfg.get("enabled", False):
+            http_proxy_url = proxycfg.get("address")
```

Comment:
is this a url or a host:port?

Reply:
That is a host:port but it'll be going away